html-style-guide.md

```diff
@@ -41,6 +41,10 @@ Aligned:
 </a>
 ```
 
+# Identification
+
+- Use `class` for identifying elements. Don't use `id`.
```

Comment:
I don't think this is really necessary. We shouldn't be styling with IDs, but there are cases where using IDs in JavaScript is more practical (and occasionally necessary, as is the case when using Google Maps).